moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_manipulation.cpp

```diff
@@ -351,7 +351,7 @@ void MotionPlanningFrame::pickObject()
 
 void MotionPlanningFrame::placeObject()
 {
-  move_group_->place(place_object_name_, place_poses_);
+  move_group_->place(place_object_name_, place_poses_, false);
```

Comment:
Please remove this again, it's not required.

Reply:
Removed